client/src/index.jsx

```diff
@@ -1,17 +1,37 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
 
-class Gallery extends Component{
+class Gallery extends React.Component{
   constructor(props){
     super(props);
     this.state = {
-      numImages: 0
-    }
-  };
+      imageCount: 0
+    };
+  }
+
+  getRequest(){
+    $.ajax({
+      url: '/restaurants',
```

Comment:
this api request should also contain the id for the restaurant.

Reply:
Thanks for catching that. I changed it to include a random restaurant id, though it's not fully functional yet:

getRequest(){
    var restID = Math.floor(Math.random()*100+1).toString().padStart(3, '0');
    $.ajax({
      url: '/restaurants/${restID}',